app/assets/stylesheets/application.scss

```diff
@@ -17,6 +17,13 @@
  *= require owl.theme
 
  */
+
+ $primary-color: #09B1D8;
+ $secondary-color: #d6dfe9;
+ $success-color: #03e53f;
+ $alert-color: #B75654;
```

Comment:
Color `#B75654` should be written as `#b75654`<br>Line should be indented 0 spaces, but was indented 1 spaces